dotcom-rendering/src/layouts/TagPageLayout.tsx

```diff
@@ -63,48 +61,6 @@ const getContainerId = (date: Date, locale: string, hasDay: boolean) => {
 	}
 };
 
-const titleStyle = css`
-	${body.medium({ fontWeight: 'regular' })};
```

Comment:
By deciding title style in the container title we get to remove all of this as this is already handled by FrontSection